tesseract_core/runtime/cli.py

```diff
@@ -151,7 +151,12 @@ def _schema_to_docstring(schema: Any, current_indent: int = 0) -> str:
 def check() -> None:
     """Check whether the Tesseract API can be imported."""
     # raises an exception if the API cannot be imported
-    get_tesseract_api()
+    api_module = get_tesseract_api()
```

Comment:
Now that we extend the scope of this function, should this also call `create_endpoints` to ensure endpoints names and schema generation works as intended?